src/drawing/drawingUtil.ts

```diff
@@ -0,0 +1,34 @@
+import { Annotation, AnnotationDrawing, PathGroup, Position, Shape } from '../@types';
+
+export function getCenter({ height, width, x, y }: Shape): Position {
+    return { x: x + width / 2, y: y + height / 2 };
+}
+
+export function getShape(pathGroups: PathGroup[]): Shape {
```

Comment:
Can this array be empty? Do things work as expected if so?

Reply:
Yeah, it'll return {0, 0, 0, 0} after I address the above comment.